dusty/daemon.py

```diff
@@ -61,6 +66,7 @@ def _listen_on_socket(socket_path, suppress_warnings):
                     suppress_warnings |= payload['suppress_warnings']
                     logging.info('Received command. fn: {} args: {} kwargs: {}'.format(fn.__name__, args, kwargs))
                     try:
+                        _refresh_warnings()
```

Comment:
Don't want it to keep saying "WARNING yo shit is out of date" when it's no longer out of date